src/main/java/org/broadinstitute/hellbender/tools/exome/allelefraction/AlleleFractionLikelihoods.java

```diff
@@ -52,33 +52,23 @@ private AlleleFractionLikelihoods() {}
      * <p>
      * where alpha = mu*beta and n = a + r
      */
-    public static double hetLogLikelihood(final AlleleFractionState state, final int segment, final AllelicCount count, final AlleleFractionIndicator indicator) {
-        return hetLogLikelihood(state, segment, count, indicator, AllelicPanelOfNormals.EMPTY_PON);
+    public static double hetLogLikelihood(final AllelicBiasParameters parameters, final double minorFraction, final AllelicCount count, final AlleleFractionIndicator indicator) {
```

Comment:
Here I think there is an argument for breaking up the state, since `hetLogLikelihood` does not need to know the full state to do its work.